collect_app/src/main/java/org/odk/collect/android/logic/AuditEvent.java

```diff
@@ -107,6 +120,18 @@ public AuditEventType getAuditEventType() {
         return auditEventType;
     }
 
+    public String getNode() {
+        return node;
+    }
+
+    public FormIndex getFormIndex() {
+        return formIndex;
+    }
+
+    public boolean newAnswerAppeared() {
```

Comment:
"appeared" implies some kind of UI-related change. Perhaps `answerChanged` or `hasNewAnswer()`?

Reply:
Done.